draft-ietf-cdni-request-routing-extensions.xml

```diff
@@ -410,33 +410,80 @@
-          <xref target="RFC8008"/>).
+      <t>The FCI Capability Advertisement Object also uses the CDNI Footprint object, as described in section 5 of
+          <xref target="RFC8008"/>. The object is built of "capability-type", "capability-value", and a "footprints"
+          array built of CDNI Footprint Objects, to specify the clients to which the capability applies.
       </t>
 
-      <t>This section specify new data-types to be used also as footprint-types,
-          extending both <xref target="RFC8006"/> and <xref target="RFC8008"/>.
+      <t>This section specify new data-types to be used by both <xref target="RFC8006"/> and <xref target="RFC8008"/>.
       </t>
 
 
       <section anchor="cdni-metadata-ipv4v6cidr-data-type" title="CDNI Metadata IPv4v6CIDR Data Type">
-        <t>Sections 4.3.5 and 4.3.6 of <xref target="RFC8006"/> specify the "IPv4CIDR" and "IPv6CIDR" footprint-types
-           (respectively), for listing different IP addresses blocks. When used as footprint constraints as part of a
-            FCI Advertisement Object, these footprint-types do not allow the combination of IPv4 and IPv6 client addresses,
-            as the constraints are additives and a client address cannot be both IPv4 and IPv6.
+        <t>As described in the CDNI Metadata interface, the Footprint Object is used as part of the LocationRule Object
+            (see section 4.2.2.1 of section <xref target="RFC8006"/>), where the LocationRule Object holds an array of
+            Footprint Objects. The action defined in the location rule should be applied on clients matching any of
+            the listed Footprint Objects: Adding a Footprint Object to the "footprints" array, extends the list of
+            clients to which the rule is applied.
         </t>
+        <t> Like the CDNI Metadata interface LocationRule, the FCI Capability Advertisement Object
+            includes an array of CDNI Footprint Objects, this time as multiple, additive footprint constraints
+            (see Appendix B of <xref target="RFC8008"/>). Meaning, the advertisement of different footprint types narrows
+            the dCDN's candidacy cumulatively. This "narrowing" semantic is different from the "extending" semantic of the
+            usage in CDN Metadata interface.
+        </t>
+
+        <t>The different semantic of the footprints arrays have significance when trying to match both IPv4 clients
+            and IPv6 clients. Sections 4.3.5 and 4.3.6 of <xref target="RFC8006"/> specify the "IPv4CIDR" and "IPv6CIDR"
+            footprint-types (respectively), for listing different IP addresses blocks. These 2 footprint-types can be
+            used together when defining a CDNI Metadata interface LocationRule. Below is an example for such a
+            LocationRule Object: The "allow" action should take place for both IPv4 clients matching subnet
```

Comment:
In general, too much wording. I don't think 8006 and the location rule is relevant here, only the FCI. you just need to explain that in FCI multiple footprints are additive constrains and therefore one cannot define a footprint that covers both ipv4 and ipv6 clients, and that's why we need a new type.